src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxResponseHeadersLength.cs

```diff
@@ -81,7 +86,13 @@ public async Task LargeSingleHeader_ThrowsException(int maxResponseHeadersLength
             },
             async server =>
             {
-                await server.HandleRequestAsync(headers: new[] { new HttpHeaderData("Foo", new string('a', handler.MaxResponseHeadersLength * 1024)) });
+                try
+                {
+                    await server.HandleRequestAsync(headers: new[] { new HttpHeaderData("Foo", new string('a', handler.MaxResponseHeadersLength * 1024)) });
+                }
+                // Ignore errors caused by the client
```

Comment:
I suggest being more descriptive why are we ignoring these
```suggestion
                // Client can respond by closing/aborting the underlying stream while we are still sending the headers, ignore these exceptions 
```

Reply:
updated.